games-action/supertuxkart/supertuxkart-0.9.1.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+inherit gnome2-utils cmake-utils eutils
+
+DESCRIPTION="A kart racing game starring Tux, the linux penguin (TuxKart fork)"
+HOMEPAGE="http://supertuxkart.sourceforge.net/"
+SRC_URI="mirror://sourceforge/supertuxkart/SuperTuxKart/${PV}/${P}-src.tar.xz
+	mirror://gentoo/${PN}.png"
+
+LICENSE="GPL-2 GPL-3 CC-BY-SA-3.0 public-domain"
+SLOT="0"
+KEYWORDS="amd64 ~ppc x86"
+IUSE="debug fribidi wiimote"
+
+RDEPEND="media-libs/libpng:0
+	media-libs/libvorbis
+	media-libs/openal
+	net-misc/curl
+	sys-libs/zlib
+	virtual/glu
+	virtual/jpeg:62
```

Comment:
I am pretty sure I messed this dependency up. Probably should be `:0`, since `:62` is only for binary-compatibility. I'll fix that afterwards.